main.tex

```diff
@@ -1781,49 +1774,23 @@
 
 \end{enumerate}
 
-\item  BEHAVOUR OF PERSONS
+\item  BEHAVIOUR OF PERSONS
 
 \begin{enumerate}[label=\arabic{enumi}.\arabic*.]
 
 \item  Behaviour of Owners, Occupiers and invitees on Common Property
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must take all reasonable steps to ensure that their invitees, do not behave in a manner likely to unreasonably interfere with the peaceful enjoyment of any other person entitled to use the Common Property.
-
-\item  An Owner or Occupier of a Lot must:
-
-\begin{enumerate}[label=(\alph*)]
+\item  An Owner or Occupier of a Lot must take all reasonable steps to ensure that their invitees do not behave in a manner likely to unreasonably interfere with the peaceful enjoyment of any other person entitled to use the Common Property.
 
-\item  when on Common Property or visible from Common Property be adequately clothed and not use language or behave in a manner likely to cause offence or embarrassment to any other person;
 \newpage
 
 
-
-\item  not hold parties or gatherings on the Common Property unless the Owners Corporation first consent in writing;
-
-\item  not consume alcohol on Common Property without first obtaining the written consent of the Owners Corporation;
-
-\item  not take glassware in any Common Property areas;
-
-\item  not smoke on any Common Property area;
-
-\item  not use skateboards, scooters, roller skates, roller blades or similar forms of recreation transport on the Common Property; and
-
-\item  ensure that any child or minor over whom the Lot Owner or Occupier has care, responsibility or control is at all times accompanied by them whilst on Common Property.
```

Comment:
These rules are already covered in the section about Reasonable Use of Common Property.